diff --git a/dist/cli/build/bundle-server.js b/dist/cli/build/bundle-server.js
index f59e677181f1d0007993e36e05901f1f012689f9..e3380f68156d1c7b29fb81422bd97915ef99f26d 100644
--- a/dist/cli/build/bundle-server.js
+++ b/dist/cli/build/bundle-server.js
@@ -100,7 +100,7 @@ export async function bundleServer(buildOpts, projectOpts) {
             // Apply updater updates, must be the last plugin
             updater.plugin,
         ],
-        external: ["./middleware/handler.mjs"],
+        external: ["./middleware/handler.mjs", "sharp", "@img/*", "*node_modules/sharp"],
         alias: {
             // When @vercel/og is not used, alias the edge entry to a throwing shim so the
             // dynamic `import("next/dist/compiled/@vercel/og/index.edge.js")` call site
